Makefile

```diff
@@ -25,6 +25,7 @@ dev.up:
 	docker-compose --project-name blockstore -f docker-compose.yml up -d
 
 dev.provision:
+	docker exec -t edx.devstack.mysql /bin/bash -c 'mysql -uroot <<< "create database blockstore_db;"'
```

Comment:
It should be possible to run `dev.provision` multiple times to update the container. Doing so is resulting in:
```
ERROR 1007 (HY000) at line 1: Can't create database 'blockstore_db'; database exists
make: *** [dev.provision] Error 1
```